CMakeLists.txt

```diff
@@ -221,14 +221,19 @@ else()
 
 endif()  # !MSVC
 
-# By default prefer STATIC build (legacy behavior)
-option(BUILD_SHARED_LIBS "Build shared libraries" OFF)
-# only expose shared option to advanced users:
+option(HWY_FORCE_STATIC_LIBS "Ignore BUILD_SHARED_LIBS" OFF)
+option(BUILD_SHARED_LIBS "Build shared libraries" ON)
```

Comment:
Please change the default to OFF so that there's no change in behavior for users.

Reply:
Done and reworked how this is applied. Now it does not change `BUILD_SHARED_LIBS`. Instead it instructs `add_library`.
Tested without options, with `-DBUILD_SHARED_LIBS=ON` and with `-DHWY_FORCE_STATIC_LIBS=ON`.

PTAL